src/handlers/ipcHandlers/facilityHandler.js

```diff
@@ -21,6 +22,13 @@ const getFacilitiesByRegion = (e, arg) => {
     });
 }
 
+const getFacilitiesByState = (e,arg) => {
+    let response = facility.getFacilityByStateId(1);
+    response.then((result) => {
+      console.log(result)
```

Comment:
Do we need to log it?

Reply:
No, but the corresponding route to integrate with the frontend doesnt exist yet. So it only ends at console.log()

Reply:
Except if you just want me to leave it empty instead.

Reply:
What if we leave it empty?

Reply:
Nothing happens which just feels wrong. Liek whats the point of making the endpoint if it just does nothing.

Reply:
True, but since this is just pending I suggest removing the logging because we don't need to log it and simply add a comment there suffixed with `TODO`

Reply:
Like `// TODO: Implement on the FE side`

Reply:
THAT LOOKS SO CRUDE AAAAA

Alright sure

Reply:
Wait, is `Handler` part of the BE in the first place?

Reply:
If this is the link between the BE and the FE and the FE hasn't even been implemented then the link shouldn't exist yet right?

Reply:
Yes it is, it is the intermediary between them. As in the event that is sent by the icpc rendered send a request to the handelr which pass it through to services.

Reply:
I see... well then, the TODO should be enough.